### PR TITLE
expanded the intro section with hello world and debugging

### LINKED ISSUE #13: Hello World

### PR DESCRIPTION
**Pull request by winkerVSbecks**
closes #13 

cc: @ssayan

### CODE REVIEW

File: book/setup/debugging.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Debugging
+
+React Native provides several tools to make debugging easier. To access the in-app developer menu:
+
+- Press `⌘ + m`
```
Comment (riavalon) [inline book/setup/debugging.md]: Is this the right command? This would just minimize the simulator wouldn't it?

Comment (winkerVSbecks) [inline book/setup/debugging.md]: You're right. That's the command on windows. I'll update the PR.

File: book/setup/hello-world.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Let's open the project in a text editor to go through the generated code.
+
+
+## Bootstrapping
+
+In order to bootstrap a React Native app we use `AppRegistry` instead of `ReactDOM`, for example:
+
+```js
+import React, { AppRegistry, Component, View, Text } from 'react-native';
```
Comment (riavalon) [inline book/setup/hello-world.md]: Importing `React` from the 'react-native' module will throw a deprecation warning, insisting to instead pull in `React` from the main react module.

File: book/setup/hello-world.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  ```sh
+  $ react-native run-ios
+  ```
+  or Open ``/Users/<userName>/reactNativeWorkshop/ios/reactNativeWorkshop.xcodeproj` in Xcode
+
+
+- Android
+
+  ```sh
+  $ react-native run-android
```
Comment (sachasayan) [inline book/setup/hello-world.md]: It's worth pointing out you can't just do this if you have a default installation of Android Studio available. You need to do a custom install with Intel Atom system images and HAXM enabled. Might be worth just linking out here: https://facebook.github.io/react-native/docs/getting-started.html

**Comment by sachasayan** [comment]
👍 with the above exceptions noted.

File: book/setup/hello-world.md (hunk 3 above)
Comment (winkerVSbecks) [inline book/setup/hello-world.md]: That's linked in the setup section already (intro/index.md)

**Comment by winkerVSbecks** [comment]
@ssayan @riavalon updated with suggested changes.

**Pull request merged**